daffodil-test/src/test/resources/org/apache/daffodil/section13/boolean/boolean.tdml

```diff
@@ -18,25 +33,52 @@
                  separatorPosition="infix" occursCountKind="implicit" emptyValueDelimiterPolicy="both"
                  representation="text" textNumberRep="standard" lengthKind="delimited" encoding="ASCII" />
 
-  <xs:element name="booleanDefault" dfdl:initiator="default1,default2,default3,default4%NL;%WSP*;">
-    <xs:complexType>
-      <xs:sequence dfdl:separator="%NL;" dfdl:separatorPosition="postfix">
-        <xs:element name="record" maxOccurs="unbounded">
-          <xs:complexType>
-              <xs:sequence dfdl:separator=','>
-                <xs:element name="default1" type="xs:boolean" default="true" dfdl:textBooleanTrueRep="true" dfdl:textBooleanFalseRep="false" />
-                <xs:element name="default2" type="xs:boolean" default="false" dfdl:textBooleanTrueRep="true" dfdl:textBooleanFalseRep="false" />
-                <xs:element name="default3" type="xs:boolean" default="1" dfdl:textBooleanTrueRep="1" dfdl:textBooleanFalseRep="0" />
-                <xs:element name="default4" type="xs:boolean" default="0" dfdl:textBooleanTrueRep="1" dfdl:textBooleanFalseRep="0" />
-              </xs:sequence>
-          </xs:complexType>
-        </xs:element>
-      </xs:sequence>
-    </xs:complexType>
-  </xs:element>
+    <xs:element name="root1" dfdl:initiator="default1,default2,default3,default4%NL;%WSP*;">
+      <xs:complexType>
+        <xs:sequence dfdl:separator="%NL;" dfdl:separatorPosition="postfix">
+          <xs:element name="record" maxOccurs="unbounded">
+            <xs:complexType>
+                <xs:sequence dfdl:separator=','>
+                  <xs:element name="default1" type="xs:boolean" default="true" dfdl:textBooleanTrueRep="true" dfdl:textBooleanFalseRep="false" />
+                  <xs:element name="default2" type="xs:boolean" default="false" dfdl:textBooleanTrueRep="true" dfdl:textBooleanFalseRep="false" />
+                  <xs:element name="default3" type="xs:boolean" default="1" dfdl:textBooleanTrueRep="1" dfdl:textBooleanFalseRep="0" />
+                  <xs:element name="default4" type="xs:boolean" default="0" dfdl:textBooleanTrueRep="1" dfdl:textBooleanFalseRep="0" />
+                </xs:sequence>
+            </xs:complexType>
+          </xs:element>
+        </xs:sequence>
+      </xs:complexType>
+    </xs:element>
 
   </tdml:defineSchema>
 
+  <!--
+  Test name: booleanDefault
+  Schema: BooleanDefaultSchema, root booleanDefault
+  Purpose: This test demonstrates parsing of valid default values
+-->
+
+  <tdml:parserTestCase name="booleanDefault" root="root1"
+                       model="BooleanDefaultSchema"
+                       roundTrip="onePass"
+                       description="Test valid boolean default values">
+    <tdml:document><![CDATA[default1,default2,default3,default4
+false,true,0,1
+]]></tdml:document>
```

Comment:
It's usually best to avoid using newlines in tests unless you're actually testing newlines. Windows vs Linux line endings can cause all kinds of headaches, and I think is causing these tests to fail.  I would reccomend replacing your ``%NL;`` and newlines with a pipe/semicolon/hash/whatever to avoid those issues.

Reply:
I don't get any failures locally.

Reply:
If you look at the "Checks" tab on the top of this PR you can see the automated tests that run for all PRs. On the windows test, the new tests you've added are failing, I suspect because of the line ending issues.